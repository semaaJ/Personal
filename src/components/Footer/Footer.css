.footer {
    text-align: center;
    font: 14px 'Inconsolata', monospace;
}

.footer hr {
    width: 250px;
}

.footerContent {
    margin: 15px;
}