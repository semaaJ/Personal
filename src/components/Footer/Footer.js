import React from 'react';
import './Footer.css';

const Footer = () => {
    return (
        <div className="footer">
            <hr />
            <div className="footerContent">
                Designed by James Miles 2020
            </div>
        </div>
    )
}   

export default Footer;