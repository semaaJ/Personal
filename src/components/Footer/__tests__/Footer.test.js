import React from 'react';
import Footer from '../Footer';
import { shallow } from 'enzyme';
import '../../../__mocks__/intersectionObserverMock';
import "../../../setupTests"

test('should render Footer correctly', () => {
    const component = shallow(<Footer />);
    expect(component.html()).toMatchSnapshot();
});
