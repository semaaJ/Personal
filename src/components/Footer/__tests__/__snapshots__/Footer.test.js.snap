// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Footer correctly 1`] = `"<div class=\\"footer\\"><hr/><div class=\\"footerContent\\">Designed by James Miles 2020</div></div>"`;
